test/cppsuite/src/main/database.h

```diff
@@ -46,10 +46,10 @@ class database {
     static std::string build_collection_name(const uint64_t id);
 
 public:
-    /*
-     * Add a new collection, this will create the underlying collection in the database.
-     */
+    /* Add a new collection, this will create the underlying collection in the database. */
```

Comment:
style nit: Our cpp style guide recommends `//` instead of `/* */` for small comments

Reply:
Updated in d771cf66ec0fd32696fd37e10f4545b5f4f82b4c